### PR TITLE
WRN-20739: Code cleanup for drawing component

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [ ] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Code cleanup for drawing component

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-20739

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian [daniel.stoian@lgepartner.com](mailto:daniel.stoian@lgepartner.com)

### CODE REVIEW

File: Drawing/Drawing.js [hunk 1]
```diff
@@ -320,23 +319,25 @@ const DrawingBase = kind({
...
+			window.localStorage.setItem('lastBrushTime', JSON.stringify(Date.now()));
+		}, []);
 
-		function setCanvasColorAndIndex (color, index) {
+		const setCanvasColorAndIndex = useCallback((color, index) => {
 			setCanvasColorValue(color);
 			setCanvasColorIndex(index);
 			window.localStorage.setItem('lastCanvasColor', JSON.stringify(color));
-		}
+			window.localStorage.setItem('lastCanvasTime', JSON.stringify(Date.now()));
```
Comment (vladut-clapou-lgp) [inline Drawing/Drawing.js:333]: This line should be deleted

File: Drawing/Drawing.js [hunk 2]
```diff
@@ -320,23 +319,25 @@ const DrawingBase = kind({
 			}
 		}, [brushColor, canvasColor, fillColor]);
 
-		function setBrushColorAndIndex (color, index) {
+		const setBrushColorAndIndex = useCallback((color, index) => {
 			setBrushColorValue(color);
 			setBrushColorIndex(index);
 			window.localStorage.setItem('lastBrushColor', JSON.stringify(color));
-		}
+			window.localStorage.setItem('lastBrushTime', JSON.stringify(Date.now()));
```
Comment (vladut-clapou-lgp) [inline Drawing/Drawing.js:326]: This line should be deleted

**Review by vladut-clapou-lgp — APPROVED** [review]
LGTM

**Review by alexandrumorariu — APPROVED** [review]
LGTM

**Pull request merged**